doc/2/api/controllers/document/export/index.md

```diff
@@ -0,0 +1,128 @@
+---
+code: true
+type: page
+title: export
+---
+
+# export <SinceBadge version="auto-version"/>
```

Comment:
Usually we put the badge under the title. Also I think this will mess up with the anchor if you put it inside title

```suggestion
# export

<SinceBadge version="auto-version"/>
```